### PR TITLE
Hello1

### PR DESCRIPTION
**Pull request by chepkok3**
Created a new branch called hello1 
Added the .gitignore file and node_modules to gitignore file.
Set up a linter for HTML and CSS.
Added descriptive README file to the project based on a given template.
Added html file that includes a header with the text "Hello Microverse!" using h1 tag .
Added css file that includes one class with styles for your header.
Added test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: .github/learn-github-actions.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14"
+      - run: npm install -g bats
+      - run: bats -v
```
Comment (anagudelogu) [inline .github/learn-github-actions.yml:13]: - [ ] While you made a great effort working on GitHub actions, the main goal of this task is to understand how to set up an HTML and CSS project. Please consider following step-by-step this [tutorial provided by Microverse](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up your linter checks on GitHub. Keep it up! 🤓 🛠️

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# Hello-Microverse-Project
\ No newline at end of file
+# Hello-Microverse-Project
+
+Title : Hello Microverse Project
+Purpose: The project is meant for learning how to use git and git.
+It aims to ensure that the following is achieved at the end of the session:
+1.Create branches with different base branches. 2. Understand the concept of version control. 3. Follow GitHub flow. 4. Open pull requests in GitHub. 5. Write descriptive, easy to understand commit messages. 6. Write short, detailed, easy to understand descriptions of pull requests. 7. Write a descriptive and easy to understand README in business English for every project submitted. 8. Demonstrate an ability to submit a project for code review.
```
Comment (anagudelogu) [inline README.md]: - [ ] Kindly use the [template provided by Microverse](https://github.com/microverseinc/readme-template) to document your code. Remember that having professional code documentation will make your projects more attractive to people that want to collaborate on them and also will provide better guidance on how to interact with them. Please consider using the template and customizing it with the project information! 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @chepkok3, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️

- [ ] You did a great job following the requirements! 💯  However, you missed some of them:

image
image

In order to achieve this, you need to add a `.gitignore` file with the contents: `test.md` and `node_modules`. Consider implementing this! 🤓 🛠️ 

_Check the comments under the review._

## Optional suggestions

- [ ] **[OPTIONAL]:** You did great with your commit messages! 💯 One personal suggestion to make them even better is to follow the guidance from this [article](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/). Your commits are going to level up! 🤓 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/learn-github-actions.yml (hunk 1 above)
Comment (chepkok3) [reply .github/learn-github-actions.yml:13]: Thank you for the good review and detailed explanations.

**Comment by chepkok3** [comment]
@anagudelogu anagudelogu I have made the recommended changes. Please review them and let me know how I have done. Thanks.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (mgmediaweb) [inline index.html]: Kindly, the previous reviewer suggested you check out the HTML/CSS tutorial, currently your site doesn't have a link to the CSS file, so you can't see the styles applied

Comment (mgmediaweb) [reply index.html]: [OPTIONAL] Kindly, I suggest you always change the title of the HTML document to an appropriate one.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @chepkok3 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost3

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (chepkok3) [reply index.html]: @mgmediaweb I just missed that. I will make the changes and thanks for noticing it.

**Comment by chepkok3** [comment]
@mgmediaweb I have made the recommended changes by adding the css link to the Html file and changed the title to a better one that is meaningful. Thanks for the feedback.

**Review by jerryowusu — APPROVED** [review]
Hi @chepkok3 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Used HTML and CSS files :heavy_check_mark: 
Correct use of github flow :+1: 

## Optional suggestions
[OPTIONAL] There are no linters set up for this project. Kindly ensure that linters are set up and working correctly to make your project conform to best standard practices. :pray: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chepkok3** [comment]
@jerryowusu I will surely work on the linters and them set up for the project. I am greatful for your review!🙏

**Pull request merged**